policies/pod/mutate_image_pull_policy_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Cruise LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//    https://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package ingress
+
+package pod
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/cruise-automation/k-rail/policies"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestPolicyImagePullPolicy(t *testing.T) {
+	ctx := context.Background()
+
+	tests := []struct {
```

Comment:
Nice test cases! It is personal preference but I want to mention it just in case: you can add descriptions to the code and executor easily by switching to a [map[string]struct{}](
https://github.com/cruise-automation/k-rail/blob/master/policies/pod/empty_dir_size_limit_test.go#L31 )